src/Agent.Worker/TestResults/FileZipper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.TestResults
+{
+    [ServiceLocator(Default = typeof(FileZipper))]
+    public interface IZipper : IAgentService
+    {
+        void CreateZipFile(string zipfileName, IEnumerable<string> files);
```

Comment:
There is only one place using this method, not sure whether worth to have it in a separate class other than a private method in the caller class.